Configurations/10-main.conf

```diff
@@ -1659,15 +1694,16 @@ my %targets = (
     },
     "darwin-i386-cc" => {
         inherit_from     => [ "darwin-common", asm("x86_asm") ],
-        cflags           => add(picker(default => "-arch i386",
-                                       release => "-fomit-frame-pointer")),
+        CFLAGS           => add(picker(release => "-fomit-frame-pointer")),
+        cflags           => add("-arch i386"),
         cppflags         => add("-DL_ENDIAN"),
         bn_ops           => "BN_LLONG RC4_INT",
         perlasm_scheme   => "macosx",
     },
     "darwin64-x86_64-cc" => {
         inherit_from     => [ "darwin-common", asm("x86_64_asm") ],
-        cflags           => add("-arch x86_64 -Wall"),
+        CFLAGS           => add("-Wall"),
```

Comment:
On *side* note this looks misplaced, in sense that it could have appeared in darwin-common. Oh well..

Reply:
Yeah...  we do need to clean this up (I've an unfinished branch that merges things like this into toolchain templates that I should make a PR from)